### DISCUSSION TITLE
Dev build breaks upon code updates

### DISCUSSION THREAD

**Discussion by Kay2dan** [blitz-js/blitz#958] [Q&A] [answered]
Hi. I am not clear on what I am doing wrong here. Almost 9 times out of 10, when I make changes in my editor, the code update breaks & I get errors. Please see this screenshot.

![Screenshot from 2020-08-29 19-27-51](https://user-images.githubusercontent.com/6276091/91643765-0e727a00-ea2e-11ea-85c0-0a298f2a25d4.png)

Refreshing the browser does not help. Its only when I rebuild by running the `blitz start` again, does it work fine. I am having to do this after every code change. As nothing of note was found when I searched among the issues section for 'hot-reload', I wonder what I am doing wrong here.

The following screenshot is when I cancel the build and rerun the `blitz start` command again. No changes to code were made.

![Screenshot from 2020-08-29 19-34-22](https://user-images.githubusercontent.com/6276091/91643839-ae300800-ea2e-11ea-8ef9-c567f31628b4.png)

My specs:

Linux Mint 19.1 Cinnamon
Node v12.8.3 (using NVM: Node Version Manager)
Tested the above on both Firefox (latest stable release) & Opera (latest stable release).
System is Lenovo Thinkpad T420  - 12GB ram (old but its gold).
Editor: VSCode (latest stable release)

I really hope this can be fixed as I am super excited to use Blitz for an MVP.

Thank you.

**Comment by flybayer** [comment]
Hmm, that's not good! Can you try upgrading react and react-dom to `0.0.0-experimental-94c0244ba` and see if that fixes it?

**Reply by Kay2dan** [reply]
Thank you Brandon, I updated to the version you spicified above. ~~however, it doesnt appear to completely. Now I am not having to restart upon every update, however, the unexpected behaviour is definitely there. I will continue the dev & see how it goes & post update here.~~
No, its still not working unfortunately. 

In the below screenshot, the browser is giving errors while the terminal is giving the all clear for the build.

![Screenshot from 2020-08-30 13-30-49](https://user-images.githubusercontent.com/6276091/91659052-103b4c80-eac5-11ea-977f-ce756749fcd4.png)

Later today, I will replicate the repo on Ubuntu system & see how that fares.

Thank you for the help.

**Reply by flybayer** [reply]
Ok, so it appears to me that this is somehow caused by your code, either a dependency issue or in your code itself.

One way to figure out the problem is to use `git bisect`. Go back and find a commit where it's working fine and use git bisect to isolate which commit is the problem.

Another way is to create a new Blitz app and then port over your code piece by piece, testing each time.

**Reply by Kay2dan** [reply]
Fair points. I will investigate & see what I can find. Will share soon. Thank you Brandon.

**Comment by Kay2dan** [comment, marked as answer]
Can the admin please close this. I reinstalled Mint OS & all packages and it appears to be working fine. Thank you.

**Reply by flybayer** [reply]
Ok, glad you got it working!
